### PR TITLE
MAM-3777-enabling-notifications-on-a-profile-returns-a-404

### PR DESCRIPTION
**Pull request by bnolens**
[MAM-3777 : Enabling notifications on a profile returns a 404](https://linear.app/theblvd/issue/MAM-3777/enabling-notifications-on-a-profile-returns-a-404)

### CODE REVIEW

File: Mammoth/Utils/UserActions.swift [hunk 1]
```diff
@@ -132,15 +132,15 @@ struct UserActions {
         let vc = NewPostViewController()
         vc.isModalInPresentation = true
         vc.fromPro = true
-        vc.proText = "@\(user.acct) "
```
Comment (rileyhCode) [inline Mammoth/Utils/UserActions.swift:135]: In general, the .acct would be preferred as it's just the @username if the mentioned account is on the same instance as the signed in user. (but of course, being correct is even more important). Was this broken?

**Review by rileyhCode — COMMENTED** [review]
I played with this, and it did seem to fix the case outlined in the ticket.

File: Mammoth/Utils/UserActions.swift (hunk 1 above)
Comment (bnolens) [reply Mammoth/Utils/UserActions.swift:135]: I believe this is broken. These are my steps:
1. I'm logged in as bnolens@staging.moth.social
2. I search for bnolens@moth.social on the discovery tab
3. I open that user's profile and hit "Mention" in the top right settings menu
4. The compose is prefilled with "@bnolens". 
5. I send that message and navigate to my profile
6. Tapping on @bnolens inside the new message navigates to bnolens@staging.moth.social and not bnolens@moth.social

**Review by rileyhCode — APPROVED** [review]
LGTM then.

**Pull request merged**